<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationsDialog</class>
 <widget class="QDialog" name="NotificationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notifications</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="closeButton">
     <property name="toolTip">
      <string>Close</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../systray.qrc">
       <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollBar" name="horizontalScrollBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="clearButton">
     <property name="toolTip">
      <string>Clear and Close</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../systray.qrc">
       <normaloff>:/images/edit-clear.png</normaloff>:/images/edit-clear.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>clearButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../systray.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>NotificationsDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>horizontalScrollBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>tabWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
